tests/wallet/sync/test_wallet_sync.py

```diff
@@ -1144,7 +1144,7 @@ async def test_dusted_wallet(
     )
     assert len(txs) == 1
     assert txs[0].spend_bundle is not None
-    await farm_wallet_node.wallet_state_manager.add_pending_transaction(txs[0])
```

Comment:
this appears to only add the *first* transaction. the new version seems to add all of them. Is this change deliberate?